Tasks/PowerShellOnTargetMachines2/task.json

```diff
@@ -0,0 +1,233 @@
+{
+    "id": "3B5693D4-5777-4FEE-862A-BD2B7A374C68",
+    "name": "PowerShellOnTargetMachines",
+    "friendlyName": "PowerShell on Target Machines",
+    "description": "Execute PowerShell scripts on remote machine(s) using Invoke-Command cmdlet",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=627414)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 3,
+        "Minor": 1,
+        "Patch": 101
```

Comment:
Seems odd that the first time you ship this version of task, the patch is already 100+. Should you reset this, when this?